src/main/java/org/jabref/gui/preferences/general/GeneralTab.java

```diff
@@ -141,4 +141,9 @@ void importTheme() {
     public void backupFileDirBrowse() {
         viewModel.backupFileDirBrowse();
     }
+
+    @FXML
+    public void openBrowser() {
+        viewModel.openBrowser();
```

Comment:
A model opening a browser seems strange. - CAn you move the `openBrowser` code from the model to here?

Reply:
Yes, for sure!

Reply:
@koppor this is done elsewhere as well
For example in the about dialog.

Reply:
Then, I'll move forward with merge - and will put an agenda item for on our DevCall agenda.